pkgs/tools/misc/czkawka/default.nix

```diff
@@ -65,6 +65,18 @@ rustPlatform.buildRustPackage rec {
     command = "czkawka_cli --version";
   };
 
+  postInstall = ''
+    # Install Icons
+    install -D data/icons/com.github.qarmin.czkawka.svg \
+      $out/share/icons/hicolor/scalable/apps/com.github.qarmin.czkawka.svg
+    install -D data/icons/com.github.qarmin.czkawka-symbolic.svg \
+      $out/share/icons/hicolor/scalable/apps/com.github.qarmin.czkawka-symbolic.svg
+
+    # Install Desktop Entry
+    install -D data/com.github.qarmin.czkawka.desktop \
+      $out/share/applications/com.github.qarmin.czkawka.desktop
```

Comment:
We could also use `makeDesktopItem`, but I see no real mentioning in `doc`.

```diff
-    install -D data/com.github.qarmin.czkawka.desktop \
-      $out/share/applications/com.github.qarmin.czkawka.desktop
+    install -Dm444 -t $out/share/applications data/com.github.qarmin.czkawka.desktop
```

Reply:
I think it's usually better to use upstream dist materials when available, and rather override any parts that won't fit. The less we need to maintain, the better.